.github/workflows/propagate_to_manifestdb.yml

```diff
@@ -0,0 +1,77 @@
+name: Propagate update to OSBuild
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  propagate:
+    runs-on: ubuntu-latest
+    env:
+      SCHUTZBOT_GH: ${{ secrets.SCHUTZBOT_GH}}
+    steps:
+      - name: Checkout OSBuild
+        uses: actions/checkout@v3
+        with:
+          path: osbuild
+          fetch-depth: 0
+
+      - name: Update Manifest-db
+        run: |
+          # update only if manifest-db SHA was updated in the last commit
+          cd osbuild
+          ACTUAL_MDB_SHA=$(git show HEAD:Schutzfile | jq -r '.global.dependencies."manifest-db".commit')
+          PREVIOUS_MDB_SHA=$(git show HEAD^:Schutzfile | jq -r '.global.dependencies."manifest-db".commit')
```

Comment:
Will is actually work? The checkout action uses `fetch-depth` == 0, so I'm not sure whether `HEAD^` is actually defined.

Reply:
Yes it does work. I've been testing it successfully.

Reply:
Actually, setting up a fetch-depth of 0 means no limitation on the fetch depth.

Reply:
I guess that `--depth` wouldn't make any sense 'cause you need at least one commit. Thanks for looking into this! :)